lang/scala/scala.py

```diff
@@ -0,0 +1,288 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+mode: user.scala
+mode: user.auto_lang
+and code.language: scala
+"""
+ctx.tags = ["user.code_operators", "user.code_generic"]
+
+# Scala Common Types
+scala_common_types = {
+    "Byte": "Byte",
+    "Int": "Int",
+    "Double": "Double",
+    "Short": "Short",
+    "Float": "Float",
+    "Long": "Long",
+    "Boolean": "Boolean",
+    "Char": "Char",
+    "String": "String",
+    "None": "None"
+}
+mod.list("scala_common_types", desc="Scala Common Types")
```

Comment:
Do you need to expose these lists to Talon? It looks like they only get used to build the code_type list.